

- CallKit
- CXErrorCodeIncomingCallError
- CXErrorCodeIncomingCallError.Code
-  CXErrorCodeIncomingCallError.Code.callUUIDAlreadyExists 

Case

# CXErrorCodeIncomingCallError.Code.callUUIDAlreadyExists

The incoming call UUID already exists.

iOS 10.0+iPadOS 10.0+Mac Catalyst 10.0+visionOS 1.0+watchOS 9.0+

``` source
case callUUIDAlreadyExists
```

## See Also

### Errors

case unknown

An unknown error occurred.

case unentitled

The app isnâ€™t entitled to receive incoming calls.

case filteredByDoNotDisturb

The incoming call is filtered because Do Not Disturb is active and the incoming caller is not a VIP.

case filteredByBlockList

The incoming call is filtered because the incoming caller has been blocked by the user.

